src/ActiveCell.tsx

```diff
@@ -100,29 +99,31 @@ class ActiveCell<Cell: Types.CellBase, Value> extends Component<
   }
 }
 
-const mapStateToProps = (state: Types.StoreState<*>) => {
+const mapStateToProps = (
+  state: Types.IStoreState<any>,
+  { DataEditor, getValue, getBindingsForCell }: Props<any, any>
+) => {
   const dimensions = state.active && getCellDimensions(state.active, state);
   if (!state.active || !dimensions) {
     return { hidden: true };
   }
   return {
     hidden: false,
     ...state.active,
-    // $FlowFixMe
     cell: Matrix.get(state.active.row, state.active.column, state.data),
     width: dimensions.width,
     height: dimensions.height,
     top: dimensions.top,
     left: dimensions.left,
-    mode: state.mode
+    mode: state.mode,
+    DataEditor,
+    getValue,
+    getBindingsForCell
   };
 };
 
-export default connect(
-  mapStateToProps,
-  {
-    setCellData: Actions.setCellData,
-    edit: Actions.edit,
-    commit: Actions.commit
-  }
-)(ActiveCell);
+export default connect(mapStateToProps, {
+  setData: Actions.setData,
```

Comment:
```suggestion
  setCellData: Actions.setCellData,
```
It's unclear why this was renamed from the original API